submissions/israelOladosu/database/transportation.sql

```diff
@@ -0,0 +1,86 @@
+CREATE DATABASE transporation;
+
+use transporation;
+
+-- classes table
+CREATE TABLE classes (
+    id INT NOT NULL AUTO_INCREMENT,
+    class_level INT NOT NULL,
+    PRIMARY KEY(id)
+);
+
+-- passangers table
+CREATE TABLE passengers (
+    id INT NOT NULL AUTO_INCREMENT,
+    firstname VARCHAR(50) NOT NULL,
+    lastname VARCHAR(50) NOT NULL,
+    sex VARCHAR(6) NOT NULL,
+    ticket_number VARCHAR(20) NOT NULL,
```

Comment:
`ticket_number` and `class_id` shouldn't be on this table. `passenger_ticket` and `passenger_class` already correspond to this on `trips` table